core/src/main/java/org/frankframework/senders/FrankSender.java

```diff
@@ -421,13 +421,13 @@ private static void setupChildSession(PipeLineSession session, ParameterValueLis
 		if (correlationId != null) {
 			childSession.put(PipeLineSession.CORRELATION_ID_KEY, correlationId);
 		}
-		childSession.put(PipeLineSession.MESSAGE_ID_KEY, MessageUtils.generateMessageId());
 		if (pvl != null) {
 			Map<String, Object> valueMap = pvl.getValueMap();
 			valueMap.remove(TARGET_PARAM_NAME);
 			valueMap.remove(SCOPE_PARAM_NAME);
 			childSession.putAll(valueMap);
 		}
```

Comment:
Dit zorgt er nu wel voor dat je hem helemaal nooit door kan geven, ook niet wanneer deze expliciet gedefineerd is als Param.

Reply:
Niet onder dezelfde session-key, inderdaad. Ik weet niet wanneer het nodig zal zijn dat de sub-adapter dezelfde `mid` session key heeft als de hoofd-adapter.
Maar als je het oorspronkelijke message-id nodig hebt dan kan je dat altijd meegeven met een parameter

`<Param name="originalMessageId" sessionKey="mid"/>`

Of zoiets.

Reply:
Tja, ik kan niet direct bedenken waar het allemaal handig zou zijn, waarschijnlijk voor foutafhandeling in een subadapter? Maar goed, voor nu maar even kijken hoe dit loopt!!